src/reducers/postReducer.js

```diff
@@ -0,0 +1,22 @@
+import post from '../data/post';
+import { ADD_POST, DELETE_POST, UPADTE_POST, UPDATE_POST } from '../actions/blogActions';
```

Comment:
`UPADTE_POST` seems to be a mistype. It's not doing anything in this file currently.